### PR TITLE
fixed /OQ30/

### PR DESCRIPTION
**Pull request by viridispirus**

### CODE REVIEW

**Comment by m1cm1c** [comment]
Not only if an exception is thrown but also if other things go wrong (like the program being killed by the OS (for example for using too much ram or whatever)) or the program calls `System.exit()`.

File: doc/Requirements Specification/Parts/Non-Functional Requirements.lyx [hunk 1]
```diff
@@ -241,8 +241,8 @@ name "/OQ30/"
 
 \end_inset
 
- Beagle can handle errors e.g.
- exceptions.
+ If the software which is measured throws an exeption, than shall Beagle
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: ...than Beagle shall...

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: if we’re on it, without any “than” at all! “If […], Beagle shall”

**Comment by acBerger** [comment]
[PDF] (http://viridispirus.github.io/Beagle/branches/OQ30/Requirements%20Specification.pdf)

it should be "an error occurs"

**Comment by jGleitz** [comment]
Sorry for the merge, I was abscent. We’ll fix it when re-reading.

**Pull request merged**